data/model/user_listening_activity.py

```diff
@@ -0,0 +1,49 @@
+import pydantic
+
+from datetime import datetime
+from typing import Optional, List
+
+
+class UserListeningActivityRecord(pydantic.BaseModel):
+    """ Each individual record for user's listening activity contains the time range,
+        timestamp for start and end of the time range and listen count.
+    """
+    time_range: str
+    from_ts: int
+    to_ts: int
+    listen_count: int
+
+
+class UserListeningActivityStatMessage(pydantic.BaseModel):
+    """ Format of messages sent to the ListenBrainz Server """
+    musicbrainz_id: str
+    type: str
+    range: str
```

Comment:
We use time_range above and range here, we should consider using time_range everywhere because range is a predefined function in python's stdlib. Also, some docs on what this field is.